pkg/sdn/plugin/node.go

```diff
@@ -38,6 +39,11 @@ import (
 	kexec "k8s.io/kubernetes/pkg/util/exec"
 )
 
+const (
+	CNI_DIR_PATH           = "/etc/cni/net.d"
```

Comment:
We don't do constants like this - if this is only used in this package, should be `camelCase`